functional-tests/test-bed/src/test/java/org/arquillian/smart/testing/ftest/configuration/DebugMode325MavenSmartTestingFunctionalTest.java

```diff
@@ -28,28 +32,32 @@
     public TestBed testBed = new TestBed(GIT_CLONE);
 
     @Test
-    public void should_show_debug_logs_when_smart_testing_is_executed_in_debug_mode() throws Exception {
+    public void should_show_debug_logs_when_smart_testing_is_executed_in_debug_mode_changes_using_325_maven()
+        throws Exception {
         // given
         final Project project = testBed.getProject();
 
         project.configureSmartTesting()
                 .executionOrder(AFFECTED)
-                .inMode(ORDERING)
-            .enable();
+                .inMode(SELECTING)
+            .enable(false);
```

Comment:
This is a bit unclear for me - flag, as well as parameter name ('install'), doesn't really explain what is the intended behaviour here. Maybe we can think of `enable(Using.EXTENSION_FILE)` or `enable(Using.SHADED_JAR)` and having `enable()` defaulted like it is now? With `enum`/strategy approach we can easily encapsulate these two different setup approaches as I hinted in the comment above.